relayer/relays/beefy/beefy-relaychain-listener.go

```diff
@@ -39,12 +38,82 @@ func NewBeefyRelaychainListener(
 func (li *BeefyRelaychainListener) Start(ctx context.Context, eg *errgroup.Group) error {
 
 	eg.Go(func() error {
+		err := li.syncBeefyJustifications(ctx, li.config.Source.Polkadot.BeefyStartingBlock, li.config.Source.SyncBlockNumberJump)
+		if err != nil {
+			return err
+		}
 		return li.subBeefyJustifications(ctx)
 	})
 
 	return nil
 }
 
+func (li *BeefyRelaychainListener) syncBeefyJustifications(ctx context.Context, startBlockNumber, syncBlockNumberJump uint64) error {
+	log.WithFields(log.Fields{"StartingBlockNumber": startBlockNumber, "SyncBlockNumberJump": syncBlockNumberJump}).Info("Syncing BEEFY justifications.")
+
+	blockNumber := startBlockNumber + syncBlockNumberJump
```

Comment:
@musnit @alistair-singh I'm not familiar with why we are allowed to skip justifications, so I think adding a comment in the code explaining that would be very useful.